packages/react-examples/src/react-charting/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.StringAxisTooltip.Example.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {
+  HorizontalBarChartWithAxis,
+  IHorizontalBarChartWithAxisProps,
+  IHorizontalBarChartWithAxisDataPoint,
+} from '@fluentui/react-charting';
+import { DefaultPalette } from '@fluentui/react/lib/Styling';
+import { ChoiceGroup, IChoiceGroupOption } from '@fluentui/react';
+
+const options: IChoiceGroupOption[] = [
+  { key: 'expandYAxisLabels', text: 'Expand Y Axis Ticks' },
+  { key: 'showTooltip', text: 'Show Tooltip at Y Axis Ticks' },
+];
+
+interface IHorizontalBarChartWithAxisState {
+  selectedCallout: string;
+}
+
+export class HorizontalBarChartWithAxisStringAxisTooltipExample extends React.Component<
+  {},
+  IHorizontalBarChartWithAxisState
+> {
+  constructor(props: IHorizontalBarChartWithAxisProps) {
+    super(props);
+    this.state = {
+      selectedCallout: 'showTooltip',
+    };
+  }
+  public render(): JSX.Element {
+    return <div>{this._basicStringAxisExample()}</div>;
+  }
+
+  private _basicStringAxisExample(): JSX.Element {
+    const points: IHorizontalBarChartWithAxisDataPoint[] = [
+      {
+        y: 'String One',
```

Comment:
There is an issue in this example. Y axis labels are not in the center of the bar

Reply:
Fixed